3-ML-Basic.sh

```diff
@@ -58,22 +57,26 @@ if [[ $(command -v conda) || (-n $CIINSTALL) ]]; then
     PIP="pip install"
 else
     execute sudo apt-get update
-    # execute sudo apt-get install python3 python3-dev python-dev python-tk -y
-    # if [[ ! -n $CIINSTALL ]]; then execute sudo apt-get install python3-pip python-pip; fi
```

Comment:
Is there any reason to keep the `update`?
I think better would be to uncomment the lines and remove the Python2 install